arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,200 @@
+        }
+        
+        super.init(coder: coder)
+        
+        // Load the geodatabase.
+        geodatabase?.load { [weak self] (error) in
+            let result: Result<Void, Error>
+            if let error = error {
+                result = .failure(error)
+            } else {
+                result = .success(())
+            }
+            self?.geodatabaseDidLoad(with: result)
+        }
+    }
+    
+    deinit {
+        if let geodatabase = geodatabase {
+            geodatabase.close()
+            try? FileManager.default.removeItem(at: geodatabase.fileURL)
+        }
+    }
+    
+    /// Called in response to the geodatabase load operation completing.
+    func geodatabaseDidLoad(with result: Result<Void, Error>) {
+        switch result {
+        case .success:
+            // Get and load the first feature table in the geodatabase.
+            featureTable = geodatabase.geodatabaseFeatureTables[0] as AGSArcGISFeatureTable
+            featureTable?.load { [weak self] error in
+                guard let self = self else { return }
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    // Create and load the feature layer from the feature table.
+                    let featureLayer = AGSFeatureLayer(featureTable: self.featureTable!)
+                    // Add the feature layer to the map.
+                    self.mapView.map?.operationalLayers.add(featureLayer)
+                    // Set the map's viewpoint to the feature layer's full extent.
+                    let extent = featureLayer.fullExtent
+                    self.mapView.setViewpoint(AGSViewpoint(targetExtent: extent!))
+                    // Add buffer graphics for the feature layer.
+                    self.queryFeatures()
+                }
+            }
+        case .failure(let error):
+            presentAlert(error: error)
+        }
+    }
+    
+    /// Creates a map with a vector tiled layer basemap.
+    func makeMap() -> AGSMap {
+        // Get the vector tiled layer by name.
+        let fillmoreVectorTiledLayer = AGSArcGISVectorTiledLayer(name: "FillmoreTopographicMap")
+        // Use the vector tiled layer as a basemap.
+        let basemap = AGSBasemap(baseLayer: fillmoreVectorTiledLayer)
+        let map = AGSMap(basemap: basemap)
+        return map
+    }
+    
+    /// Create buffer graphics for the features.
+    func queryFeatures() {
+        // Create the query parameters.
+        let queryParameters = AGSQueryParameters()
+        // Set the where clause to filter for buffer sizes greater than 0.
+        queryParameters.whereClause = "BufferSize > 0"
+        // Create an array of graphics to add to the graphics overlay.
+        var graphics = [AGSGraphic]()
+        // Query the features with the query parameters.
+        featureTable?.queryFeatures(with: queryParameters) { [weak self ] result, error in
```

Comment:
```suggestion
        featureTable.queryFeatures(with: queryParameters) { [weak self ] result, error in
```
Remove ❓ if the suggestion above is adopted.